src/MainWindow.vala

```diff
@@ -214,11 +214,48 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
 
         spinner = new Gtk.Spinner ();
 
+        var automatic_updates_button = new Granite.SwitchModelButton (_("Automatic Updates")) {
+            description = _("Automatically install updates to curated apps")
```

Comment:
If we exclude non-paid-for paid app, this might work:

```suggestion
            description = _("Automatically update free and paid-for curated apps in the background")
```